src/firefly/js/tables/FilterInfo.js

```diff
@@ -158,6 +158,7 @@ export class FilterInfo {
             const parts = conditions.split(COND_SEP);
             for (let i = 0; i < parts.length; i += 2) {
                 const [cname, op, val] = parseInput(parts[i]);
+                if (isNaN(Number(val))) return false;
```

Comment:
I see the problem. However, strings like this is slipping though `>5 xxx <6`.
In this case op is `>` and val is `5 xxx <6`.

What do you think of this?
_remove line 161 and Insert after line 158._
```js
if (parts.length===1 && parts[0].match(/ [A-Za-z]\S* /)) return false;
```
It is not perfect but it will catch a lot of error of this type.

Reply:
I don't think this is very useful because it doesn't prevent `> 5 and >5 xxx <6`
yet, it will wrongly prevent this `LIKE '% m31 %'` condition.

Reply:
ok, I will work on the function some more.